misk-aws/src/main/kotlin/misk/dynamodb/FakeDynamoDbModule.kt

```diff
@@ -0,0 +1,73 @@
+package misk.dynamodb
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.client.builder.AwsClientBuilder
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper
+import com.amazonaws.services.dynamodbv2.document.DynamoDB
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput
+import com.amazonaws.services.dynamodbv2.model.ResourceInUseException
+import com.google.inject.Provides
+import misk.cloud.aws.FakeAwsEnvironmentModule
+import misk.inject.KAbstractModule
+import javax.inject.Singleton
+import kotlin.reflect.KClass
+
+internal class FakeDynamoDbModule(
+  private val entities: List<Pair<String, KClass<*>>>
+) : KAbstractModule() {
+  override fun configure() {
+    install(FakeAwsEnvironmentModule())
+  }
+
+  @Provides @Singleton
+  fun providesAmazonDynamoDB(): AmazonDynamoDB {
+    val dynamoDb = AmazonDynamoDBClientBuilder
```

Comment:
Would you consider returning a `DynamoDBEmbedded.create().amazonDynamoDB()` instead of the docker one?

Reply:
@mightyguava do you have a preference?

Reply:
I don’t have a preference. But for sake of expediency I don’t think it should block this PR. Since you got embedded dynamo working why don’t we merge this pr and you can submit a follow up?

Reply:
I prefer embedded over docker (faster, easier to debug, easier to support, etc) but I also agree that it shouldn't be blocking the PR.

Reply:
deleted code